pootle/apps/pootle_fs/response.py

```diff
@@ -0,0 +1,128 @@
+    "description":
+        "Stores updated where filesystem version was new or newer"}
+FS_RESPONSE["added_from_pootle"] = {
+    "title": "Added from Pootle",
+    "description":
+        "Files staged from Pootle that were new or newer than their files"}
+FS_RESPONSE["pruned_from_fs"] = {
+    "title": "Pruned from filesystem",
+    "description":
+        ("Files removed from the filesystem that did not have matching Pootle "
+         "Stores")}
+FS_RESPONSE["fetched_from_fs"] = {
+    "title": "Fetched from filesystem",
+    "description":
+        ("Files staged from the filesystem that were new or newer than their "
+         "Pootle Stores")}
+FS_RESPONSE["pushed_to_fs"] = {
+    "title": "Pushed to filesystem",
+    "description":
+        "Files updated where Pootle Store version was new or newer"}
+FS_RESPONSE["staged_for_removal"] = {
+    "title": "Staged for removal",
+    "description":
+        ("Files or Stores staged for removal where the corresponding "
+         "file/Store is missing")}
+FS_RESPONSE["removed"] = {
+    "title": "Removed",
+    "description":
+        ("Files or Stores removed that no longer had corresponding files or "
+         "Stores")}
+FS_RESPONSE["staged_for_merge_fs"] = {
+    "title": "Staged for merge (FS Wins)",
+    "description":
+        ("Files or Stores staged for merge where the corresponding "
+         "file/Store has also been updated or created")}
+FS_RESPONSE["staged_for_merge_pootle"] = {
+    "title": "Staged for merge (Pootle Wins)",
+    "description":
+        ("Files or Stores staged for merge where the corresponding "
+         "file/Store has also been updated or created")}
+FS_RESPONSE["merged_from_fs"] = {
+    "title": "Merged from fs",
+    "description":
+        ("Merged - FS won where unit updates conflicted")}
+FS_RESPONSE["merged_from_pootle"] = {
+    "title": "Merged from pootle",
+    "description":
+        ("Merged - Pootle won where unit updates conflicted")}
+
+
+class ProjectFSItemResponse(ItemResponse):
+
+    def __str__(self):
+        return (
+            "<%s(%s)%s: %s %s::%s>"
+            % (self.__class__.__name__,
+               self.response.context,
+               self.failed and " FAILED" or "",
+               self.action_type,
+               self.pootle_path,
+               self.fs_path))
+
+    @property
+    def failed(self):
+        return not self.complete
+
+    @property
+    def fs_path(self):
+        return self.fs_state.fs_path
+
+    @property
+    def fs_state(self):
+        return self.kwargs["fs_state"]
+
+    @property
+    def pootle_path(self):
+        return self.fs_state.pootle_path
+
+    @cached_property
+    def store(self):
+        if self.fs_state.store:
+            return self.fs_state.store
+        if self.store_fs:
+            return self.store_fs.store
+        try:
+            return Store.objects.get(
+                pootle_path=self.fs_state.pootle_path)
+        except Store.DoesNotExist:
```

Comment:
in relation to "just paths related"

imagine the following scenario

- file  appears - ie `fs_untracked`
- user diidnt actually want the file (dupes or something)
- user removes 
- we then have an action(/response) that has no `Store`, and no `StoreFS` - the only thing it has is a an `fs_path`, and a corresponding `pootle_path` for where it would have been put had it been kept